recipes/repository.rb

```diff
@@ -25,6 +25,13 @@
     action :install
   end
 
+  # Trust new APT key
+  execute 'install new apt key' do
+    command 'apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 A2923DFF56EDA6E76E55E492D3A80E30382E94DE'
+    not_if 'apt-key list | grep 382E94DE'
+  end
+
```

Comment:
Hmm we could use the resource twice (as long as we name the 2 differently, otherwise we're cloning the resource, which is a Bad thing to do), but that would define 2 `.list` files with the same repository, which `apt` doesn't like (it will spew out warnings because the repo is duplicated).

And it wouldn't look good either. That said I'll look into it a bit more to see if we couldn't make use of the logic that's already implemented in the `apt` cookbook.